libs/mtesitoo-backend-api/src/main/java/com/mtesitoo/backend/model/Seller.java

```diff
@@ -80,6 +80,10 @@ public void setmDescription(String mDescription) {
         this.mDescription = mDescription;
     }
 
+    public void setmThumbnail(String thumbnail) {
```

Comment:
I know this is auto-generated setter method. Personally, that 'm' in `setmThumbnail` bugs me ...haha... 🤣

Reply:
Yeah it does to me too 🤣  I'm not particularly fond of Hungarian notation in general...